packages/@okta/vuepress-site/docs/guides/add-an-external-idp/configure-idp-in-okta/index.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Create an Identity Provider in Okta
+---
+To connect your org to the Identity Provider, add and configure that Identity Provider in Okta.
+
+1. From the Developer Console, hover over **Users** and then select **Social & Identity Providers** from the menu that appears. If you are using the Classic UI, hover over **Security** and then select **Identity Providers**.
```

Comment:
For a user that doesn't have access to the developer console, do they think of the admin UI as the `Classic UI`? Should we call it the `Admin Console`? Is there another universal name for that part of Okta?

Reply:
@ZoeMonosson-okta - I was going by how the UI labels it - which is "Classic UI". I know that it's considered the admin console, but that's not what the UI call it. We should open a ticket to get that changed so that what people call it matches what they select to switch to that view?? Meanwhile, I'll switch the text to call it the admin console.

Reply:
@susanharper-okta I'd be happy to open a ticket to rethink the "Classic UI" label in the UI itself because I don't think it's very clear. 

However, what I was trying to highlight is that if a user has access to the dev console, I'd assume that they would add an idp through the dev console. If they don't have access to the dev console, they would use the admin console. If they only have access to the admin console, they would never see the admin dropdown called "Classic UI." If we are assuming that all users making an idp have access to the dev console, then this feedback is irrelevant :)